### ISSUE TITLE
Check if Vanilla Land's textures should really be disabled

### ISSUE THREAD

**Issue by Tyler799** [Tyler799/Morrowind-2019#74]
We need to do a quality/visual check of if those textures that are told to be disabled should really be disabled. It's just a holdover from the 2017 guide.

**Comment by Darkloke** [comment]
Vanilla Land Textures are 2048x2048 vs 1024x2014 in Lush Ascadian Isles.

They also intersect with ones from MVP and Tyddys Landscape Textures and since they are all different its may be a matter of taste what to use.

**Comment by Darkloke** [comment]
posting just in case from a close/dublicate issue:

"There's a lot of overlap between these two mods, but 50 textures or so from Tyddy's aren't covered by Vanilla Land."

Hm, actually as i see Vanilla Land covers ALL textures from Tyddy's landscape (assuming u installed HQ+UHQ Packs, BUT Tyddy's UHQ pack offers textures in 4k resolution vs 2k from Vanilla.

So i suppose its enough to have Vanilla Land + UHQ Tyddy's landscape over.

**Comment by Tyler799** [comment]
When I check in MO2, I can see around 43 textures not covered by vanilla land that Tyddy does. Though there are 10 total textures in VL that are disabled in the guide, so those could be part of that sum. 

I got vanilla land from here: https://www.nexusmods.com/morrowind/mods/45953?tab=files

What is this "HQ + UHQ pack"?

**Comment by Darkloke** [comment]
Tyddy's pack have 162 textures
Vanilla land have 166 textures

But UHQ pack from Tyddy'd offers 50 textures in 4k (vs 2k in vanilla).
HQ and UHQ packs are from Tyddy's Landscape Textures 
http://www.fullrest.ru/files/tyd_landscape-texture_compilation_1/files

**Comment by Tyler799** [comment]
The number of texture isn't a useful or meaningful number:

With Tyddy's placed before Vanilla, I get this conflict result: https://imgur.com/M1gRPa0 (42 files that Tyddy's provides that *no other mod* provides after it)

As for Tyddy's, I see what you mean now. After installing HQ + UHQ, I go down to 11 textures remaining that Tyddy touches that Vanilla Land does not. Not too many. 

As for what is better to use, I think I'm still going to leave it as an open choice. While I prefer high resolution textures, I've found dozens of occasions where I saw a better-looking, but ultimately lower-resolution texture.
